src/Feature/Configuration/Feature_Configuration.re

```diff
@@ -333,8 +333,22 @@ module Commands = {
     );
 };
 
+module Keybindings = {
+  open Feature_Input.Schema;
+
+  let command = Commands.openConfigurationFile.id;
+
+  let openConfigOnMac =
+    bind(~key="<D-,>", ~condition="isMac" |> WhenExpr.parse, ~command);
+
+  let openConfigOnOther =
+    bind(~key="<C-,>", ~condition="!isMac" |> WhenExpr.parse, ~command);
```

Comment:
I was worried this might conflict with a default Vim keybinding, but doesn't appear that it does 👍